docs/low-code-spark/configuration.md

```diff
@@ -9,76 +9,103 @@ tags:
   - config
 ---
 
-Allows you to define configurations to control various aspects of your Pipeline. Prophecy IDE allows you to define three kinds of configurations:
+Allows you to define configurations to control various aspects of your Pipeline.
+![Config Option](img/config-option.png)
 
-1. **Pipeline Configuration**: name-value pair per [Fabric](/concepts/fabric.md) which can then be accessed in Pipeline as `${name}`. <br/> Eg: `for Fabric = dev, SOURCE_PATH: dbfs:/dev/file.csv`, <br/> `for Fabric = prod, SOURCE_PATH: dbfs:/prod/file.csv`
-   :::note
-   Each name-value pair has to be first defined in the `Common` tab. This can then be overridden in the individual Fabric tabs.
-   ![Configurations - Common](./img/configs_common.png)
-   :::
-2. **Spark Configuration**: Runtime Spark configurations as name-value pairs.
-   :::note
-   The name-value pairs will be set inside the Spark runtime configurations as `spark.conf.set(name, value)`
-   ![Configurations - Spark](./img/configs_spark.png)
-   This will be compiled as:
+Prophecy IDE allows you to define three kinds of configurations:
 
-   ```python
-   spark.conf.set("spark.sql.autoBroadcastJoinThreshold", "10485760")
-   ```
+## Spark Configuration
 
-   :::<br/>
+Runtime Spark configurations as name-value pairs. The name-value pairs will be set inside the Spark runtime configurations as `spark.conf.set(name, value)`
 
-3. **Hadoop Configuration**: Hadoop configurations as name-value paris.
-   :::note
-   The name-value pairs will be set inside the Hadoop configuration as `spark.sparkContext.hadoopConfiguration.set(name, value)`
-   ![Configurations - Spark](./img/configs_hadoop.png)
-   This will be compiled as:
+![Configurations - Spark](./img/configs_spark.png)
 
-   ```python
-   spark.sparkContext.hadoopConfiguration.set("fs.s3a.access.key", "my_access_key")
-   ```
+## Hadoop Configuration
 
-   :::<br/>
+Hadoop configurations as name-value pairs. The name-value pairs will be set inside the Hadoop configuration as `spark.sparkContext.hadoopConfiguration.set(name, value)`
 
----
+![Configurations - Spark](./img/configs_hadoop.png)
+
+## Pipeline Configuration
+
+Config values which can be set at Pipeline level and then be accessed inside any component in the Pipeline. [Multiple instances](#pipeline-configuration-instances)
+of configuration can be created per Pipeline.
+![Configurations - Common](img/config-pipeline-eg1.png)
+
+Syntax for using configuration inside Gems:
```

Comment:
Can we turn this into a visual example with a Code block? We might have to break this up based on the use case a bit more. The user won't understand what "visual language SQL" vs "visual language Scala" means.